zebra-network/src/isolated.rs

```diff
@@ -124,6 +124,9 @@ where
 /// Transactions sent over this connection can be linked to the sending and receiving IP address
 /// by passive internet observers.
 ///
+/// TODO: Refactor [`connect_isolated_run_tor`](tor::connect_isolated_run_tor)
+/// below since the reference doesn't exist.
+///
 /// Prefer [`connect_isolated_run_tor`](tor::connect_isolated_run_tor) if available.
```

Comment:
The function was renamed:

```suggestion
/// Prefer [`connect_isolated_tor`](tor::connect_isolated_tor) if available.
```